### PR TITLE
[9.0][IMP] stock_inventory_exclude_sublocaton: track visibility and only editable in draft state

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Adds a dependency with `stock_inventory_chatter` to track the `exclude_sublocation` field in the chatter. Also taking the occasion to fix the readonly states.

### CODE REVIEW

File: stock_inventory_exclude_sublocation/__openerp__.py [hunk 1]
```diff
@@ -6,12 +6,12 @@
...
     "summary": "Allow to perform inventories of a location without including "
                "its child locations.",
-    "version": "9.0.1.0.0",
+    "version": "9.0.1.0.1",
     "author": "Eficent,"
               "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-warehouse",
     "category": "Warehouse Management",
-    "depends": ["stock"],
+    "depends": ["stock", "stock_inventory_chatter"],
```
Comment (aheficent) [inline stock_inventory_exclude_sublocation/__openerp__.py]: Why do you remove this?

**Review by AaronHForgeFlow — APPROVED** [review]
:+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by mreficent** [comment]
Added minor fixes from other two modules just to avoid to make two other PRs.

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: cool

**Comment by LoisRForgeFlow** [comment]
:+1:

**Comment by MiquelRForgeFlow** [comment]
Added some minor fixes for others modules to solve some warnings. They are minor fixes, so I preferred to group here them than just make several PRs.

**Comment by MiquelRForgeFlow** [comment]
@lreficent , you should edit the title as your initial commit has been changed. You can't change the dependencies of an already merged module. So, we should add this dependency when migrating to v10

**Comment by LoisRForgeFlow** [comment]
@mreficent let's wait then to v10 to do the fix. The main purpose of this PR is lost then, do you want me to close this one so you can open a new PR for the other changes you have proposed?

**Pull request closed without merging**